openmp/cmake/OpenMPTesting.cmake

```
@@ -65,6 +65,11 @@ else()
   else()
     set(OPENMP_FILECHECK_EXECUTABLE ${LLVM_RUNTIME_OUTPUT_INTDIR}/FileCheck)
   endif()
+  if (NOT TARGET "clang")
+    message(STATUS "Cannot find 'clang'.")
+    message(WARNING "The check targets will not be available!")
+    set(ENABLE_CHECK_TARGETS FALSE)
```

```suggestion
    set(ENABLE_CHECK_TARGETS FALSE)
    set(OPENMP_ENABLE_LIBOMPTARGET OFF)
```

@shiltian would you accept someting like this (with an amended warning)?